src/Illuminate/Support/Str.php

```diff
@@ -253,6 +253,23 @@ public static function quickRandom($length = 16)
     }
 
     /**
+     * Replace a given value in the string sequentially with an array.
+     *
+     * @param  string  $search
+     * @param  array   $replace
```

Comment:
This would reduce consistency with the rest of Str.php and helpers.php files.
I think it would be better to change code style in whole files at once, and in a dedicated commit.